docs/rules/enforce-memoize-getters.md

```
@@ -0,0 +1,45 @@
+
+```ts
+import { Memoize } from '@blumintinc/typescript-memoize';
+class Example {
+  @Memoize()
+  private get fetcher() {
+    return createFetcher(); // Stable instance
+  }
+}
+```
+
+## Notes
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)